test/e2e/template/NgxFlickingTemplate.ts

```diff
@@ -0,0 +1,24 @@
+import { FlickingOptions } from "../../../src";
+import { NgxFlickingComponent } from "../../../packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component";
```

Comment:
If testing locally, it seems to be important whether to e2e after deploying native or before deploying native.


If you test before deployment, you may need a way to temporarily change the local version of the framework.

Reply:
That's right. That's why I've talked about dependency injection before.

Reply:
Maybe it will be resolved with resolve
https://webpack.js.org/configuration/resolve/

If not, there seems to be a way to install it from a local address.